typings/index.d.ts

```diff
@@ -2664,6 +2670,25 @@ declare module 'discord.js' {
 
   type GuildMemberResolvable = GuildMember | UserResolvable;
 
+  interface GuildMembershipScreening {
+    description: string;
+    enabled: boolean;
+    formFields: GuildMembershipScreeningField[];
+  }
+
+  interface GuildMembershipScreeningData {
+    description?: string;
+    enabled?: boolean;
+    formFields?: GuildMembershipScreeningField[];
+  }
```

Comment:
```suggestion
  type GuildMembershipScreeningData = Partial<GuildMembershipScreening>;
```
either this or just directly use `Partial<GuildMembershipScreening>`